lib/peatio/ranger.rb

```diff
@@ -75,11 +62,26 @@ def handle(msg)
       end
     end
 
-    def handshake(handshake)
+    def handshake(hs)
       @client = Peatio::MQ::Events::Client.new(@socket)
-      query = URI::decode_www_form(handshake.query_string)
-      subscribe(query.map{ |item| item.last if item.first == "stream" })
+
+      query = URI::decode_www_form(hs.query_string)
+      subscribe(query.map {|item| item.last if item.first == "stream"})
       @logger.info "ranger: WebSocket connection openned"
+
+      if hs.headers.key?("Authorization")
```

Comment:
Ambassador injects `authorization` Header not `Authorization` which is strange a bit for me
So I think we need to support both